index.html

```diff
@@ -7189,7 +7189,51 @@ <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
         </p>
       </section>
     </section>
-
+    <section class="informative">
+    <h3>Language maps</h3>
+    The following is an example credential that demonstrates support for multiple languages and base directions.
+           <pre class="example nohighlight"
+             title="A sample credential that demonstrates multiple languages">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
+  "issuer": {
+    "id": "https://university.example/issuers/565049",
+    "name": "Example University",
+    "description": "A public university focusing on teaching examples."
+  },
+  "validFrom": "2015-05-10T12:30:00Z",
+  "name": "Example University Degree",
+  "description": "2015 Bachelor of Science and Arts Degree",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "ExampleBachelorDegree",
+      "name": [
+        {
+          "@value": "Bachelor of Science and Arts",
+          "@language": "en-us"
+        },
+        {
+          "@value": "Baccalauréat en sciences et arts",
+          "@language": "fr",
```

Comment:
```suggestion
        {
          "@value": "License en sciences et arts",
          "@language": "fr-FR",
```

Welcome to the world of internationalization and localization!

***In France***, this would be a translation error. Indeed, the term "Baccalauréat" is the degree you get at the age of cca. 18 after finishing your middle school studies. I believe this is roughly what you call "High school diploma" in the US, and "A-level" in the UK. The French term "License" is for the diploma you get after three years of studies at a University, which, I believe, is what the Bachelor degree is in the US.

However, as far as I know, the terms are different in Belgium ("bachelier"), Switzerland ("bachelor"). To make all this even nicer, the term "Baccalauréat" is actually the the correct term in… Quebec. Hence my change to "fr-FR" if we want to stick to France.  Alternatively, we can keep the term as is, but use "fr-CA". (Or should we have both just for the fun of it?)